test/unit/org/apache/cassandra/config/DatabaseDescriptorRefTest.java

```diff
@@ -67,16 +67,29 @@ public class DatabaseDescriptorRefTest
     "org.apache.cassandra.audit.BinAuditLogger",
     "org.apache.cassandra.audit.BinLogAuditLogger",
     "org.apache.cassandra.audit.IAuditLogger",
+    "org.apache.cassandra.auth.AbstractCIDRAuthorizer",
```

Comment:
@tiagomlalves  this seems like a lot. I think that by reverting that change as you suggested, we would get rid of this. We can target these changes in another ticket if you insist on changing that.

Reply:
I agree -  I reverted this change. My goal was to check if I could still get the CI green (which I managed). I'm not sure if there's value having these changes in another ticket.